com.syntevo.SmartSynchronize.json

```diff
@@ -0,0 +1,84 @@
+{
+  "app-id": "com.syntevo.SmartSynchronize",
+  "command": "smartsynchronize",
+  "runtime": "org.gnome.Platform",
```

Comment:
Tried to do the same that I did in `SmartGit` with this one, but I'm getting this beautiful stacktrace at startup:

```shell
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]: Error invoking SmartSynchronize
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]: java.lang.reflect.InvocationTargetException
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at java.base/java.lang.reflect.Method.invoke(Method.java:564)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at com.syntevo.QBootLoader.main(SourceFile:102)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]: Caused by: java.lang.UnsatisfiedLinkError: org.eclipse.swt.internal.gtk.OS._cachejvmptr()V
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.internal.gtk.OS._cachejvmptr(Native Method)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.internal.gtk.OS.cachejvmptr(OS.java:486)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:97)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:134)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:80)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at org.eclipse.swt.widgets.Display.<clinit>(Display.java:142)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at smartsynchronize.hK.a(SourceFile:62)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at smartsynchronize.xH.a(SourceFile:108)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at smartsynchronize.KM.a(SourceFile:120)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at smartsynchronize.yd.a(SourceFile:67)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         at SmartSynchronize.main(SourceFile:11)
Aug 20 21:28:09 uplink com.syntevo.SmartSynchronize.desktop[31734]:         ... 5 more
```